base/util.go

```diff
@@ -139,6 +139,10 @@ func ToInt64(value interface{}) (int64, bool) {
 	return 0, false
 }
 
+func IsPseudoDoc(ID string) bool {
+	return strings.HasPrefix(ID, "_user/") || strings.HasPrefix(ID, "_role/")
```

Comment:
I think it would be better to avoid the performance overhead of two strings.HasPrefix checks on every changes entry we send.  It might be better to set a (new) flag on the changesEntry struct when we create the _user entry in the first place.